config/routes.rb

```diff
@@ -5,6 +5,7 @@
   ActiveAdmin.routes(self)
   get '/prizes', to: 'prizes#index'
   post '/prizes', to: 'prizes#create'
+  get '/socials', to: 'companies#socials_from_company'
```

Comment:
Not a big fan of this, tbh. As a rule of thumb, especially as you're getting started, try to only use RESTful actions. For example, instead of `/socials` going to the `companies` controller you can do `/companies/:id/socials`, or in your case, since you're using the `selected` attribute, you can do `/socials` map to `socials#show` or `selected_company_socials#show`. Don't be afraid to have many controllers.